frontend/packages/dev-console/src/components/health-checks/__tests__/create-health-check-probe-data.ts

```diff
@@ -0,0 +1,494 @@
+          },
+        },
+        resources: {},
+        successfulBuildsHistoryLimit: 5,
+        failedBuildsHistoryLimit: 5,
+        strategy: {
+          type: 'Source',
+          sourceStrategy: {
+            from: {
+              kind: 'ImageStreamTag',
+              namespace: 'openshift',
+              name: 'python:3.6',
+            },
+          },
+        },
+        postCommit: {},
+        source: {
+          type: 'Git',
+          git: {
+            uri: 'https://github.com/divyanshiGupta/nationalparks-py',
+          },
+          contextDir: '/',
+        },
+        triggers: [
+          {
+            type: 'Generic',
+            generic: {
+              secretReference: {
+                name: 'nationalparks-py-generic-webhook-secret',
+              },
+            },
+          },
+          {
+            type: 'GitHub',
+            github: {
+              secretReference: {
+                name: 'nationalparks-py-github-webhook-secret',
+              },
+            },
+          },
+          {
+            type: 'ImageChange',
+            imageChange: {
+              lastTriggeredImageID:
+                'image-registry.openshift-image-registry.svc:5000/openshift/python@sha256:dde8883b3033d9b3d0e88b8c74304fba4b23cd5c07a164e71ee352b899a7803e',
+            },
+          },
+          {
+            type: 'ConfigChange',
+          },
+        ],
+        runPolicy: 'Serial',
+      },
+      status: {
+        lastVersion: 3,
+      },
+    },
+  },
+  imageStream: {
+    loaded: true,
+    loadError: '',
+    data: [
+      {
+        kind: 'ImageStream',
+        apiVersion: 'image.openshift.io/v1',
+        metadata: {
+          annotations: {
+            'app.openshift.io/vcs-ref': 'master',
+            'app.openshift.io/vcs-uri': 'https://github.com/divyanshiGupta/nationalparks-py',
+          },
+          selfLink: '/apis/image.openshift.io/v1/namespaces/div/imagestreams/nationalparks-py',
+          resourceVersion: '676247',
+          name: 'nationalparks-py',
+          uid: '2bc985ac-f834-45e5-9a86-830edb6bc8bd',
+          creationTimestamp: '2020-01-15T15:51:47Z',
+          generation: 1,
+          namespace: 'div',
+          labels: {
+            app: 'nationalparks-py',
+            'app.kubernetes.io/component': 'nationalparks-py',
+            'app.kubernetes.io/instance': 'nationalparks-py',
+            'app.kubernetes.io/name': 'python',
+            'app.kubernetes.io/part-of': 'nodejs-rest-http-app',
+            'app.openshift.io/runtime': 'python',
+            'app.openshift.io/runtime-version': '3.6',
+          },
+        },
+        spec: {
+          lookupPolicy: {
+            local: false,
+          },
+        },
+        status: {
+          dockerImageRepository:
+            'image-registry.openshift-image-registry.svc:5000/div/nationalparks-py',
+        },
+      },
+    ],
+  },
+};
+
+export const healthChecksData: HealthChecksData = {
```

Comment:
Same as above.

Reply:
Why don't you create a central file for all data related to health checks. Just import and use whatever you already have available from other data files and add extra properties for health checks that you need.